java/src/main/java/ai/rapids/cudf/ParquetWriterOptions.java

```diff
@@ -56,6 +57,26 @@ public Builder withTimestampInt96(boolean int96) {
       return this;
     }
 
+    /**
+     * Overwrite flattened precision values for all decimal columns that are expected to be in
```

Comment:
It would be good to mention that it's an in-order traversal of _all_ decimal columns, whether they are top-level columns in the table or nested within another column type like list or struct columns.  Bonus points if the example shows two decimal columns within the struct column, showing the relative ordering when that nested type is flattened.